### PR TITLE
API SDK 2.1 - Authorization re-construction and Expose V1 APIs.

### PR DESCRIPTION
**Pull request by bunnyc1986**
Note: 
* This includes a commit to update CSharp templates.
* The SDK version number is pending. We need a decision `2.1.0` vs `3.0.0`
* Include V1 APIs

### CODE REVIEW

**Comment by bunnyc1986** [comment]
The impact to individual language can be previewed:
* C# https://github.com/square/connect-csharp-sdk/pull/21
* PHP https://github.com/square/connect-php-sdk/pull/10
* Ruby https://github.com/square/connect-ruby-sdk/pull/7
* Python https://github.com/square/connect-python-sdk/pull/13/files

**Comment by bunnyc1986** [comment]
Ignore the branch name for now. It's going to be `api-2.x` branch.

File: swagger-config/config-csharp.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "packageName": "Square.Connect",
-  "packageVersion": "2.0.2",
+  "packageVersion": "3.0.0",
```
Comment (bunnyc1986) [inline swagger-config/config-csharp.json]: these will be changed to `2.1.0`

**Comment by frojasg** [comment]
I don't think this is relevant anymore

**Pull request closed without merging**